### PR TITLE
Add artDialog@6.0.4 w/ git auto-update

### LINKED ISSUE #10221: [Request] Add artDialog
Library name: https://github.com/aui/artDialog
Git repository url: https://github.com/aui/artDialog
npm package url(optional): https://github.com/aui/artDialog
License(s): LGPL
Official homepage: https://github.com/aui/artDialog
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by qwe19272375**
Pull request for issue: #10221 
Related issue(s): #10159

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/aui/artDialog
 * Official website (optional, not the repository): https://aui.github.io/artDialog/
 * NPM package url (optional):
 * License and its reference: LGPL-3.0, [ref](https://github.com/aui/artDialog/blob/master/LICENSE.md)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 281
   - Count of stars: 2380
   - Count of forks: 1016
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #10221, cc @aui

### CODE REVIEW

**Comment by qwe19272375** [comment]
@pvnr0082t Please help me review this pull request, thank you.

File: ajax/libs/artDialog/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "artDialog",
+  "filename": "dialog-min.js",
+  "version": "6.0.4",
+  "description": "经典的网页对话框组件，内外皆用心雕琢",
```
Comment (pvnr0082t) [inline ajax/libs/artDialog/package.json]: @qwe19272375 Please translate it into English.

File: ajax/libs/artDialog/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "homepage": "https://aui.github.io/artDialog/",
+  "keywords": [
+    "dialog",
+    "ui",
+    "jquery"
```
Comment (pvnr0082t) [inline ajax/libs/artDialog/package.json]: Where did you get `"ui"`, and `"jquery"`? I don't think these are good keywords.

File: ajax/libs/artDialog/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  ],
+  "author": "aui",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aui/artDialog.git"
+  },
+  "license": "LGPL",
```
Comment (pvnr0082t) [inline ajax/libs/artDialog/package.json]: This is not precise. You should take a look at [tools/license-list.json](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json).

File: ajax/libs/artDialog/package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/aui/artDialog.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "dialog*"
```
Comment (pvnr0082t) [inline ajax/libs/artDialog/package.json]: We can use `"**/*"` under dist directory.

**Comment by qwe19272375** [comment]
Translated `description`. Change license to `LGPL-2.1` and use `**/*` in dist directory.
Modify new keyword to `"dialog"`, `"bubble"`, and `"artDialog"`.
@pvnr0082t Please help me review this PR again, thank you.

File: ajax/libs/artDialog/package.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "artDialog",
+  "filename": "dialog-min.js",
+  "version": "6.0.4",
+  "description": "classical web dialog module which perfectly crafted from inside to outside",
+  "homepage": "https://aui.github.io/artDialog/",
+  "keywords": [
+    "dialog",
+    "bubble",
```
Comment (pvnr0082t) [inline ajax/libs/artDialog/package.json]: Why do you use bubble as keyword? Any relate with this lib?

**Comment by qwe19272375** [comment]
Add `#10159` to related issue. Update license to `LGPL-3.0` ([license-ref](https://github.com/aui/artDialog/issues/292)).
Modify new keyword to `"dialog"`, `"artDialog"`, and `"modal"`. [ref](http://aui.github.io/artDialog/doc/index.html#quickref-modal)
@pvnr0082t Please help me review this PR, and please give me some advice if there are other better keywords, thanks!

File: ajax/libs/artDialog/package.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "artDialog",
+  "filename": "dialog-min.js",
+  "version": "6.0.4",
+  "description": "classical web dialog module which perfectly crafted from inside to outside",
+  "homepage": "https://aui.github.io/artDialog/",
+  "keywords": [
+    "dialog",
+    "modal",
```
Comment (pvnr0082t) [inline ajax/libs/artDialog/package.json]: It's not a good keyword I think...Let's use HTML5 instead.

**Comment by qwe19272375** [comment]
Modify new keyword to `"dialog"`, `"artDialog"`, and `"html5"`.
@pvnr0082t Please help me review this pull request, thanks!  😃

**Comment by qwe19272375** [comment]
Commit message updated.
@pvnr0082t Please help me review this pull request, thanks! 😃

**Comment by PeterDaveHello** [comment]
@qwe19272375 @pvnr0082t there is v7 on npm, please discuss with its author about if that can be tagged/released on GitHub.

**Comment by aui** [comment]
@PeterDaveHello @qwe19272375 Have added a tag, thank you guys

**Pull request merged**